routes/categories.js

```diff
@@ -0,0 +1,25 @@
+const getCategories = require("../mvc/controllers/getCategories.js");
+const express = require("express");
+const router = express.Router();
+
+router.route("/").get(getCategories);
+
+router
+  .route("/*")
+  .get((req, res) => {
+    res
+      .status(404)
+      .send({ msg: "Path Not Found. The correct path is /api/categories" });
+  })
+  .post((req, res) => {
+    res.status(404).send({ msg: "Permission denied! Cannot post there." });
+  })
+  .put((req, res) => {
```

Comment:
This should be a patch, not put, please!

Reply:
Fixed :)